Add documentation for xk6-disruptor (#891)
grafana/k6-docs: src/data/markdown/docs/40 xk6-disruptor/03 Examples.md

---
title: 'Examples'
excerpt: 'Examples of how to use the xk6-disruptor extension to introduce faults in k6 tests.'
---

> In the next sections we present some examples of how to use the `xk6-disruptor` extension to introduce faults in `k6` tests. Some examples make use the [k6-kubernetes](http://github.com/grafana/xk6-kubernetes) extension. Ensure the custom `k6` binary you are using is built with both `xk6-disruptor` and `xk6-kubernetes` extensions. See the [Installation guide](../01-get-started/03-installation.md) for details.

Reviewer: ```suggestion
In these sections we present some examples of how to use the `xk6-disruptor` extension to introduce faults in `k6` tests.
Some examples use the [k6-kubernetes](http://github.com/grafana/xk6-kubernetes) extension.
So, to follow, ensure the custom `k6` binary you are using is built with both `xk6-disruptor` and `xk6-kubernetes` extensions. See the [Installation guide](../01-get-started/03-installation.md) for details.
```

Author: Not sure about "these" instead of "the next". I thought "these" always refereed to something previously mentioned.

> Also, check your test environment satisfies the requirements described in the [get started guide](../01-get-started/02-requirements.md). In particular, check you have the credentials for accessing the Kubernetes cluster used for the test properly configured and that this cluster is configured for exposing your application using an external IP.

Reviewer: ```suggestion
Also check that your test environment satisfies the requirements described in the [get started guide](../01-get-started/02-requirements.md).
In particular, check that:
- You have the properly configured the credentials to access the Kubernetes cluster used for the test 
- This cluster is configured to expose your application using an external IP.
```
